archive_viewer/mixins/file_io.py

```diff
@@ -0,0 +1,254 @@
+from os import getenv
+from re import compile
+from typing import Tuple
+from pathlib import Path
+from datetime import (datetime, timedelta)
+from urllib.parse import urlparse
+from qtpy.QtWidgets import (QMessageBox, QFileDialog)
+from config import (logger, save_file_dir)
+from av_file_convert import ArchiveViewerFileConverter
+
+
+class FileIOMixin:
+    def file_io_init(self):
```

Comment:
can you add documentation to this method?

Reply:
Solved in commit 3f00fff13322325cd1f81101ea9b124ff61f02fb